source/overview/vpat.rst

```diff
@@ -0,0 +1,481 @@
+
+- **Level of Support & Supporting Features**: SUPPORTS 
+
+- **Remarks and Explanations**: Nothing to add. 
+
+(f) Textual information shall be provided through operating system functions for displaying text. The minimum information that shall be made available is text content, text input caret location, and text attributes.
+
+- **Level of Support & Supporting Features**: TBD 
+
+- **Remarks and Explanations**: TBD
+
+(g) Applications shall not override user selected contrast and color selections and other individual display attributes.
+
+- **Level of Support & Supporting Features**: SUPPORTS THROUGH EQUIVALENT FACILITATION
+
+- **Remarks and Explanations**: Mattermost web application can be used in high contrast mode with support for use selected colors and contrast options. 
+
+(h) When animation is displayed, the information shall be displayable in at least one non-animated presentation mode at the option of the user.
+
+- **Level of Support & Supporting Features**: NOT APPLICABLE  
+
+- **Remarks and Explanations**: No core functionality in the product relies on animations. While some loading indicators are animated, failure to load is documented in text with error messages. 
+
+(i) Color coding shall not be used as the only means of conveying information, indicating an action, prompting a response, or distinguishing a visual element.
+
+- **Level of Support & Supporting Features**: SUPPORTS
+
+- **Remarks and Explanations**: No indicators rely on color alone. 
+
+(j) When a product permits a user to adjust color and contrast settings, a variety of color selections capable of producing a range of contrast levels shall be provided.
+
+- **Level of Support & Supporting Features**: SUPPORTS
+
+- **Remarks and Explanations**: See `full documentation <https://docs.mattermost.com/help/settings/theme-colors.html>`_.
+
+(k) Software shall not use flashing or blinking text, objects, or other elements having a flash or blink frequency greater than 2 Hz and lower than 55 Hz.
+
+- **Level of Support & Supporting Features**: SUPPORTS 
+
+- **Remarks and Explanations**: Nothing to add. 
+
+(l) When electronic forms are used, the form shall allow people using Assistive Technology to access the information, field elements, and functionality required for completion and submission of the form, including all directions and cues.
+
+- **Level of Support & Supporting Features**: TBD 
+
+- **Remarks and Explanations**: TBD
+
+Section 1194.22 Web-based Intranet and Internet information and Applications - Detail 
+----------------------------------------------------------------------------------------
+
+(a) A text equivalent for every non-text element shall be provided (e.g., via "alt", "longdesc", or in element content).
+
+- **Level of Support & Supporting Features**: TBD 
```

Comment:
Mostly supported. I think we might be missing a few.  Basically just need tooltips for images.